src/modulation/gfsk.rs

```diff
@@ -0,0 +1,15 @@
+//! Common GFSK modulation options
+
+/// Basic GFSK channel configuration
+#[derive(Clone, PartialEq, Debug)]
+#[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
+pub struct GfskChannel {
+    /// Channel frequency in kHz
+    pub freq_khz: u32,
+
+    /// Channel bandwidth in kHz
```

Comment:
This needs to be hertz, at least for the sx1231 most RxBw options are not aligned with kHz.

Reply:
Also, is this receiver bandwidth or transmitter frequency deviation?

Reply:
good question, i was aiming for modulation bandwidth (ie. frequency deviation) as ime this is how channels are usually described / receiver bandwidth is typically driven by this